### DISCUSSION TITLE
Add Default Option for Select style fields

### DISCUSSION THREAD

**Discussion by dougblackjr** [ExpressionEngine/ExpressionEngine#3632] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**

Select fields (select dropdown, multiselect, checkboxes, radio buttons) have multiple options available for users. It would be awesome to be able set a default for these when creating them. This way users could have a default when creating content in the CP UI.

**Describe the solution you'd like**
Simple field to set a default option in the UI for the field during field creation.

**Describe alternatives you've considered**
Unfortunately, can't address this inside of the UI without creating a custom field.

- [X] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
@dougblackjr I like the idea in general, but I'm not sure how this could be implemented UX-wise, considering there are multiple ways to provide options to select field, including dynamic (e.g. populate from other entries). So when creating a field, there's no value to choose default from yet

**Reply by dougblackjr** [reply]
I think even a text box would be fine, just some way to choose a default.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4797 (still need to finetune it though)
